velox/exec/Spill.h

```diff
@@ -463,19 +464,27 @@ struct SpillPartitionId {
   /// advance the partition start bit when go to the next level of recursive
   /// spilling.
   bool operator<(const SpillPartitionId& other) const {
-    if (partitionBitOffset != other.partitionBitOffset) {
-      return partitionBitOffset > other.partitionBitOffset;
+    if (partitionBitOffset_ != other.partitionBitOffset_) {
+      return partitionBitOffset_ > other.partitionBitOffset_;
     }
-    return partitionNumber < other.partitionNumber;
+    return partitionNumber_ < other.partitionNumber_;
   }
 
   bool operator>(const SpillPartitionId& other) const {
     return (*this != other) && !(*this < other);
   }
 
   std::string toString() const {
-    return fmt::format("[{},{}]", partitionBitOffset, partitionNumber);
+    return fmt::format("[{},{}]", partitionBitOffset_, partitionNumber_);
   }
+
+  uint8_t partitionBitOffset() const { return partitionBitOffset_; }
+
+  int32_t partitionNumber() const { return partitionNumber_; }
+
+ private:
+  uint8_t partitionBitOffset_{0};
```

Comment:
Can these be const? Why do we need to initialize these? Don't we do that in the constructor?

Reply:
I need to sort it in test and might need in production code. This is already a data class which can't be modified.